cdap-ui/app/cdap/components/DataPrep/Directives/ExtractFields/UsingDelimiterModal/index.js

```diff
@@ -102,110 +112,77 @@ export default class SplitColumnDirective extends Component {
         });
       });
   }
-
-  renderCustomDelimiter() {
-    if (this.state.delimiterSelection !== 'Custom Separator') { return null; }
-
-    return (
-      <div className="custom-delimiter-input">
-        <input
-          type="text"
-          className="form-control mousetrap"
-          onChange={this.handleDelimiterInput}
-          onKeyPress={this.handleKeyPress}
-          value={this.state.customDelimiter}
-          placeholder="e.g. \d+"
-          ref={ref => this.customDelimiterRef = ref}
-        />
-      </div>
-    );
-  }
-
-  renderDetail() {
-    if (!this.props.isOpen) { return null; }
-
+  render() {
     const OPTIONS = [
```

Comment:
Since you're working on this file already, can we make this small change? Instead of using `OPTIONS`, we can just use `Object.keys(DELIMITER_MAP)`, so we don't have to repeat these option values again.